pages/dictionary/search_history.tsx

```diff
@@ -3,10 +3,22 @@ import styles from '>>/styles/dictionary.module.css'
 
 
 type Props = {
-    searchWord: Map<string, number>
+    setDisplayText: React.Dispatch<React.SetStateAction<string>>
+    searchList: String[]
 }
 
-const History = ({ searchWord }: Props) => {
+const History = ({ setDisplayText, searchList }: Props) => {
+    const jumpTo = (word) => {
+        setDisplayText(word);
+    }
+
+    const searchWord = searchList.map((searchText, index) => {
```

Comment:
Rename to searchedWords
```suggestion
    const searchedWords = searchList.map((searchText, index) => {
```

Reply:
Fixed in [[fd7af37](https://github.com/cherryshek-1014/react-projects/pull/1/commits/fd7af37fc7ed7b98e1e3bc3d55b96ac146b789b8)](url)